spikeextractors/extractors/yassextractors/yassextractors.py

```diff
@@ -0,0 +1,55 @@
+from spikeextractors import SortingExtractor
+from spikeextractors.extractors.numpyextractors import NumpyRecordingExtractor
+import numpy as np
+from pathlib import Path
+from spikeextractors.extraction_tools import check_valid_unit_id
+
+HAVE_YASS = True
+
+class YassSortingExtractor(SortingExtractor):
+
+    extractor_name = 'YassExtractor'
+    installed = HAVE_YASS  # check at class level if installed or not
+    is_writable = True
+    mode = 'file'
+    installation_mesg = ""
+    
+    
+    def __init__(self, file_path):
+        SortingExtractor.__init__(self):
+        
+        self.fname_spike_train = file_path
+        self.spike_train = np.load(self.fname_spike_train)
+        #self._sampling_frequency = my_sampling_frequency
```

Comment:
@catubc samplig_frequency is not required, but highly recommended. Can you load that too?

Reply:
yes